app/admin/admin_users.rb

```diff
@@ -0,0 +1,28 @@
+ActiveAdmin.register AdminUser do
+  permit_params :email, :password, :password_confirmation
+
+  index do
```

Comment:
Just for the knowledge, Can you please explain me know, That which kind of work we can achieve using this code?

Reply:
-done